tests/calico_cni_k8s_test.go

```diff
@@ -65,25 +86,31 @@ var _ = Describe("Kubernetes CNI tests", func() {
 	if err != nil {
 		panic(err)
 	}
+	config, err := clientcmd.DefaultClientConfig.ClientConfig()
+	if err != nil {
+		panic(err)
+	}
+	k8sClient, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
 
 	BeforeEach(func() {
-		testutils.WipeK8sPods()
-		testutils.WipeEtcd()
+		testutils.WipeDatastore()
 
 		// Create the node for these tests. The IPAM code requires a corresponding Calico node to exist.
-		var err error
-		n := v3.NewNode()
-		n.Name, err = names.Hostname()
+		var name string
```

Comment:
think we can just do a `name, err := names.Hostname()` here instead of predeclaring.